### ISSUE TITLE
Add page size and current page index to data browser

### ISSUE THREAD

**Issue by badboy-tian** [parse-community/parse-dashboard#2571]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [ ] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [ ] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Current Limitation

can not set pagesize, so sometimes load slow
### Feature / Enhancement Description

add pagesize and pageindex in the bottom of the page
### Alternatives / Workarounds

no
### 3rd Party References

<img width="1139" alt="image" src="https://github.com/parse-community/parse-dashboard/assets/2835129/80231b96-3d62-45f4-9d30-1a2cfe4a5412">

<img width="1312" alt="image" src="https://github.com/parse-community/parse-dashboard/assets/2835129/b6efe99b-18cc-445a-be45-e2416f9866a7">

**Comment by mtrezza** [comment]
Good idea! What is the 3rd party ref? Looks like a simple clean design.

**Comment by badboy-tian** [comment]
@mtrezza This is a Chinese version of parse-server, maybe. But it does not support self-built servers, and the minimum consumption is 30 yuan per day.  https://www.leancloud.cn/
![image](https://github.com/parse-community/parse-dashboard/assets/2835129/b39e8c36-db37-4726-9693-7f206fd4d1d3)

**Comment by mtrezza** [comment]
I see, yes, we are aware of this service.

**Comment by Studycode001** [comment]
Can I add this feature plzz

**Comment by alokdangre** [comment]
Can I add this feature please?
@badboy-tian

**Comment by badboy-tian** [comment]
yes, you can

Studycode001 ***@***.***>于2024年7月12日 周五13:47写道：

**Comment by JiteshSinghShekhawat** [comment]
is this issue still open ? @badboy-tian @mtrezza

**Comment by badboy-tian** [comment]
@JiteshSinghShekhawat  yes...

**Comment by JiteshSinghShekhawat** [comment]
Thanks for confirming! Could you please assign this issue to me so I can start working on it?

**Comment by badboy-tian** [comment]
@JiteshSinghShekhawat i do not know how to assign this issue to you.

**Comment by badboy-tian** [comment]
@mtrezza perhaps need help. assign this issue to @JiteshSinghShekhawat

**Comment by JiteshSinghShekhawat** [comment]
@mtrezza  will you assign me the issue ?

**Comment by mtrezza** [comment]
Anyone please feel free to pick up the issue. Please manage among yourselves in case you'd like to collaborate and divide the work (and bounty). I'd recommend to just post a comment that you're picking this up for others to be aware of it.

**Comment by JiteshSinghShekhawat** [comment]
I'm picking up this issue and will be working on it.

**Comment by mtrezza** [comment]
Great, please open a PR linked to this issue to indicate that this is currently being developed and keep the PR up-to-date. Without PR, or if the PR goes stale for some time, we assume the issue is not actively being developed and may be picked up by someone else.

**Comment by mtrezza** [comment]
I believe this issue could be a good workaround until infinite scrolling is fixed with https://github.com/parse-community/parse-dashboard/issues/1551 via https://github.com/parse-community/parse-server/issues/7637.

**Comment by mtrezza** [comment]
@vardhan0604, @muhleder Would you be interested in working on this? Dashboard currently has no scrolling feature which is really limiting the data browser use.

**Comment by vardhan0604** [comment]
> [@vardhan0604](https://github.com/vardhan0604), [@muhleder](https://github.com/muhleder) Would you be interested in working on this? Dashboard currently has no scrolling feature which is really limiting the data browser use.

Hey @mtrezza , I’m currently a bit busy, but I’ll try to take a look at the issue as soon as I can. I agree that adding a scrolling feature would greatly improve the data browser's usability. I’ll keep you updated once I dive into it. Thanks!

**Comment by alokdangre** [comment]
Hi @mtrezza,

I'd like to take a stab at adding the scrolling feature to the dashboard. I understand this is a key improvement for the data browser’s usability, and I'm confident I can make the necessary enhancements. Could you please assign this issue to me? I'll make sure to keep everyone updated as I work through it.

Thanks!

**Comment by mtrezza** [comment]
@alokdangre sure, please go ahead; we don't actually "assign" issues, but if you submit a PR, we'll be happy to review it

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.1.0-alpha.1](https://github.com/parse-community/parse-dashboard/releases/tag/7.1.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.1.0](https://github.com/parse-community/parse-dashboard/releases/tag/7.1.0)
